### DISCUSSION TITLE
Mermaid mindmap support?

### DISCUSSION THREAD

**Discussion by karl-johan-grahn** [community/community#35689] [Other Feature Feedback, Questions, & Ideas] [answered]
# Description

When will support for mermaid mindmap be added? I have tried to locate the version of mermaid that GitHub is using, but was not able to find it. Mindmap support was added with https://github.com/mermaid-js/mermaid/pull/3410

## Reproduction Steps

The mindmap on this page is not rendered:
https://github.com/mermaid-js/mermaid/blob/develop/docs/mindmap.md

## Fault Type

Permanent - Occurring repeatably

## Blocking?

Yes

## Logs

```
Error rendering embedded code

Parse error on line 1:
mindmap root((mind
^
Expecting 'open_directive', 'NEWLINE', 'SPACE', 'GRAPH', got 'ALPHA'

For more information, see https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams
```

**Comment by pwandzil** [comment, marked as answer]
Support is already there!

**Reply by karl-johan-grahn** [reply]
Correct, I can see it rendered now at https://github.com/mermaid-js/mermaid/blob/develop/docs/syntax/mindmap.md
